fs.go

```diff
@@ -16,6 +15,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Marshaler is the interface implemented by types that
+// can marshal themselves into valid TOML.
+// TODO(carolynvs) Consider adding this to go-toml?
+type Marshaler interface {
```

Comment:
Probably better not to export this as an interface, unless we have to. As your TODO notes, if it's going to be exported, that should be done by go-toml. If we can get away with exporting, `marshaler`, or better `tomlMarshaler` would be preferable.